app/src/main/java/org/gdg/frisbee/android/Const.java

```diff
@@ -39,20 +39,21 @@ private Const() {
     public static final String EXTRA_TAGGED_EVENT = "org.gdg.frisbee.TAGGED_EVENT";
     public static final String EXTRA_TAGGED_EVENT_CACHEKEY = "org.gdg.frisbee.TAGGED_EVENT_CACHEKEY";
     //Special Events Dates
-    public static final DateTime START_TIME_IOEXTENDED =    new DateTime(2015, 5, 1, 0, 0);
-    public static final DateTime START_TIME_STUDY_JAMS =    new DateTime(2015, 2, 1, 0, 0);
+    public static final DateTime START_TIME_DEVFEST =       new DateTime(2015, 9, 1, 0, 0);

```

Comment:
I guess it is displaying study jam events due to the starting date of DevFest, is it true?